src/rp2/plugin/accounting_method/hifo.py

```diff
@@ -28,10 +29,10 @@
 # HIFO plugin. See https://www.investopedia.com/terms/h/hifo.asp. This plugin uses universal application, not per-wallet application:
 # this means there is one queue for each coin across every wallet and exchange and the accounting method is applied to each such queue.
 # More on this at https://www.forbes.com/sites/shehanchandrasekera/2020/09/17/what-crypto-taxpayers-need-to-know-about-fifo-lifo-hifo-specific-id/
-class AccountingMethod(AbstractAccountingMethod):
+class AccountingMethod(AbstractHeapAccountingMethod):
```

Comment:
Change the comment at line 42 in positive terms, similar to what we did for FIFO (i.e.: "This loop features...").

Reply:
Done